packages/components/src/components/form/Autocomplete/Autocomplete.tsx

```diff
@@ -16,6 +16,21 @@ import {
 
 const [AutocompleteProvider, useAutocompleteContext] = createContext<AutocompleteContextProps>({});
 
+const stateReducer = (state, actionAndChanges) => {
+  const { type, changes } = actionAndChanges;
+  switch (type) {
+    case useCombobox.stateChangeTypes.InputFocus:
+    case useCombobox.stateChangeTypes.InputChange:
+      // this prevents the menu from opening when there is no input value
+      return {
+        ...changes, // default Downshift new state changes on item selection.
+        isOpen: Boolean(changes.inputValue), // keep the menu open if there is a value.
+      };
+    default:
+      return changes; // otherwise business as usual.
+  }
+};
+
```

Comment:
@wescosta was this required by product/design?

Reply:
Yes. As per @rob-w-tyrrell request, we want to show the autocomplete suggestion only when the person has typed at least 1 character.

Reply:
Ok, design had a different opinion about that when the component was implemented. Not sure if we want to add a property to control that behaviour

Reply:
Ok. We can check with Rob and Hans tomorrow.

Reply:
[added an optional expectedCharLengthToOpen prop to Autocomplete component](https://github.com/cmpsr/composer/pull/785/commits/05f886bdf4f1d0540f754ac4c096f994f898de22). Thus we can keep the original behavior while allowing to only show results when the user has typed something.